components/picture_selector.tsx

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState, useEffect} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {localizeMessage} from 'utils/utils';
+import * as FileUtils from 'utils/file_utils';
+
+import './picture_selector.scss';
+
+type Props = {
+    src?: string;
+    defaultSrc?: string;
+    loadingPicture?: boolean;
+    onSelect: (file: File) => void;
+    onRemove: () => void;
+};
+
+const PictureSelector: React.FC<Props> = (props: Props) => {
+    const {src, defaultSrc, loadingPicture, onSelect, onRemove} = props;
+
+    const [image, setImage] = useState<string>();
```

Comment:
We need to pass an initial value to `useState` so we avoid undefined on initial render.

Reply:
We want this to be undefined if there is no image present. Worse thing that happens is that the `div` using the `image` variable won't have a background image.